crates/re_components/Cargo.toml

```
@@ -0,0 +1,78 @@
+[package]
+name = "re_components"
+authors.workspace = true
+description = "The standard rerun data types, component types, and archetypes"
```

```suggestion
description = "The standard Rerun data types, component types, and archetypes"
```

kinda curious that only `archetype` is a single word, but I'd say that's correct